radis/levels/partfunc_cdsd.py

```diff
@@ -272,9 +274,17 @@ def __init__(
         self.use_cached = use_cached
         self.levelsfmt = levelsfmt
         self.viblvl_label = viblvl_label
+        self.last_modification = time.ctime(
+            getmtime(getTestFile(r"co2_cdsd_hamiltonian_fragment.levels"))
+        )
+        print("Here is the last mod {0}".format(self.last_modification))
```

Comment:
this should be removed in the final version. (or hide it behind a `if verbose>=2` or something